### PR TITLE
Simpler smooth algorithm

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by arnaudon** [comment]
needs https://github.com/BlueBrain/diameter-synthesis/pull/26

**Comment by arnaudon** [comment]
I will clean that one once diameter-synthesis is released

**Comment by arnaudon** [comment]
Coverage increased, but it's still not happy... what a pain in the *ss! :P

File: src/morphology_workflows/tasks/repair.py [hunk 1]
```diff
@@ -256,6 +256,6 @@ class PlotSmoothDiameters(SkippableMixin(True), ElementValidationTask):
     def inputs(self):
         """ """
         return {
-            RepairNeurites: {"morph_path": "morph_path"},
+            FixZeroDiameters: {"morph_path": "morph_path"},
```
Comment (adrien-berchet) [inline src/morphology_workflows/tasks/repair.py:259]: Why changing this?

Comment (arnaudon) [reply src/morphology_workflows/tasks/repair.py:259]: so it's consistent with the SmoothDiameter task, this was a mistake from previous PR

Comment (adrien-berchet) [reply src/morphology_workflows/tasks/repair.py:259]: Ah ok

**Pull request merged**